tests/coverage/coverage.py

```diff
@@ -62,12 +62,14 @@ def to_dict(self):
 class TestCoverage:
     file_coverage: dict[str, FileTestCoverage]
     test_count: int
+    num_bad_tests: int
     num_covered_function_variants: int
     total_function_variants: int
 
     def __init__(self, ext_uris):
         self.file_coverage = dict()
         self.test_count = 0
+        self.num_bad_tests = 0
```

Comment:
Is a bad test something that had a catastrophic failure or does it merely mean that the result is not what it should be?  Consider using another name.

Reply:
Fixed it